app/src/main/java/com/example/grosharies/ui/theme/Theme.kt

```diff
@@ -0,0 +1,35 @@
+package com.example.grosharies.ui.theme
+
+import androidx.compose.foundation.isSystemInDarkTheme
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.darkColors
+import androidx.compose.material.lightColors
+import androidx.compose.runtime.Composable
+
+private val DarkColorPalette = darkColors(
```

Comment:
I'd have probably made a colors_dark and colors_light (like the file you defined the colors in) So it's more isolated.